CONTRIBUTING.md

```diff
@@ -67,10 +67,8 @@ env ASDF_DIRENV_DEBUG=true  bats -x test/use_asdf.bats -f 'use multiple versions
 
 ## Getting help
 
-Your pull-request needs not to be perfect. If you feel like you've implemented your core idea
+Your pull-request does not need to be perfect. If you feel like you've implemented your core idea
```

Comment:
> needs not to be perfect

I imagine we're okay with a perfect PR, haha. It's just that we're also okay with PRs that are not perfect.